scripts/gceworker.sh

```diff
@@ -24,38 +24,31 @@ function start_and_wait() {
     retry gcloud compute ssh "${1}" --command=true || true
 }
 
-case "${cmd}" in
-    gcloud)
-    gcloud "$@"
-    ;;
-    create)
+function createVM() {
     if [[ "${COCKROACH_DEV_LICENSE:-}" ]]; then
       echo "Using dev license key from \$COCKROACH_DEV_LICENSE"
     else
       echo -n "Enter your dev license key (if any): "
-      read COCKROACH_DEV_LICENSE
+      read -r COCKROACH_DEV_LICENSE
     fi
 
-    gsuite_account_for_label="$(gcloud auth list | \
-        grep '^\*' | \
-        sed -e 's/\* *//' -e 's/@/__at__/g' -e 's/\./__dot__/g'\
-        )"
-    gcloud compute instances \
-           create "${NAME}" \
-           --machine-type "n2-custom-24-32768" \
-           --network "default" \
-           --maintenance-policy "MIGRATE" \
-           --image-project "ubuntu-os-cloud" \
-           --image-family "ubuntu-2004-lts" \
-           --boot-disk-size "250" \
-           --boot-disk-type "pd-ssd" \
-           --boot-disk-device-name "${NAME}" \
-           --scopes "cloud-platform" \
-           --labels "created-by=${gsuite_account_for_label:0:63}"
+    gsuite_account_for_label="$(gcloud auth list --filter=status:ACTIVE --format='value(account)' | sed -e 's/@/__at__/g' -e 's/\./__dot__/g')"
+    gcloud compute instances create "${NAME}" \
+      --machine-type "n2-custom-24-32768" \
+      --network "default" \
+      --maintenance-policy "MIGRATE" \
+      --image-project "ubuntu-os-cloud" \
+      --image-family "ubuntu-2004-lts" \
+      --boot-disk-size "250" \
+      --boot-disk-type "pd-ssd" \
+      --boot-disk-device-name "${NAME}" \
+      --scopes "cloud-platform" \
+      --labels "created-by=${gsuite_account_for_label:0:63}" \
+      --metadata=startup-script-url=https://raw.githubusercontent.com/cockroachdb/cockroach/master/build/bootstrap/refresh-autoshutdown.cron.sh
```

Comment:
The consequences of designing this this way will be that we won't be able to rename or change the behavior of `build/bootstrap/refresh-autoshutdown.cron.sh` and `build/bootstrap/autoshutdown.cron.sh`; they must always exist and must do basically the same thing. Maybe this is OK, but is something that you should keep in mind before committing to structuring the code this way.

Reply:
Good point. Maybe it's not a big deal (that we cannot rename the file easily) - let me ponder about this. 🤔

Reply:
If you are going to keep it that way, maybe it would be best to put a big comment at the beginning of both scripts that says "DO NOT RENAME OR DELETE THIS FILE" and explaining the situation.